CMakeLists.txt

```diff
@@ -11,8 +11,14 @@ endif()
 
 include(sources.cmake)
 
+if(POLICY CMP0077)
+  cmake_policy(SET CMP0077 NEW) #option does nothing when a normal variable of the same name exists.
```

Comment:
Please add that this policy was introduced in CMAKE 3.13 so we can removed once the minimum requirement reaches that point.

Reply:
BTW, this CMakeLists.txt requires cmake2.8+, did anyone tested it works on cmake recently? Is that part of some CI somewhere?

Reply:
Yes, it's tested in CentOS, which installs CMake 2.8 by default.